_drafts/2017-05-16-lingtypology.Rmd

```diff
@@ -0,0 +1,199 @@
+```{r}
+sign_df <- data.frame(languages = c("American Sign Language", "Russian Sign Language", "French Sign Language"),
+                      popup = c("https://goo.gl/La2Oab", "https://goo.gl/lYVYbx", "https://goo.gl/YRsqva"))
+```
+Then we need to change the links to HTML codes:
+```{r}
+sign_df$popup <- paste0("<video width='200' height='150' controls> <source src='",
+                  as.character(sign_df$popup),
+                  "' type='video/mp4'></video>")
+```
+Now we can create a map:
+```{r}
+map.feature(sign_df$languages,
+            popup = sign_df$popup)
+```
+
+### 3.4 Contour plot
+It is possible to add isoglosses to a map using contour plots that are based on a kernel density estimation.  
+I will use a built-in dataset of Circassian villages in Russia that contains the genealogical affiliation of the language spoken in the respective villages, their geographical coordinates and the names of the districts in which they are located. Now let's divide the villages into two regions based on the branch of Circassian that is spoken.
+```{r}
+map.feature(circassian$language,
+            features = circassian$language,
+            popup = circassian$village,
+            latitude = circassian$latitude,
+            longitude = circassian$longitude,
+            color = c("darkgreen", "navy"),
+            density.estimation = TRUE)
+```
+
+It is possible to change what is treated as the same area using `density.longitude.width` and `density.latitude.width` arguments.
+
+```{r}
+map.feature(circassian$language,
+            features = circassian$language,
+            longitude = circassian$longitude,
+            latitude = circassian$latitude,
+            color = c("darkgreen", "navy"),
+            density.estimation = T,
+            density.longitude.width = 2,
+            density.latitude.width =0.8)
+```
+
+This example shows that it is also possible to add your own coordinates to the `map.feature()`.
+
+## 4. Conclusions
+In this post I showed the main fucntions of `lingtypology` for creating interactive linguistic maps, but there are a lot of other functions that make the visualizations more flexible, so take a look at the [manual page](https://ropensci.github.io/lingtypology/) for more information on functionalities. 
+If you have any suggestions, please create an issue on [GitHub](https://github.com/ropensci/lingtypology/issues).
+
+
+## 5. Acknowledgments
+I would like to thank [Misha Daniel](https://www.hse.ru/en/org/persons/6635821), [Samira Verhees](https://www.hse.ru/en/staff/samiraverhees), [Calle Börstell](https://borstell.github.io/#name), [Timo Roettger](http://phonetik.phil-fak.uni-koeln.de/troettger.html) and [Robert Forkel](https://www.shh.mpg.de/person/45369/25522) for their ideas and comments. I sincerely thank the reviewers for their constructive criticism and valuable comments during the rOpenSci reviewing.
```

Comment:
not a problem re: not using the Tweeter
fyi, here are links to 3 tweets that went to nearly 13,000 followers of rOpenSci. Hopefully getting your package some attention!

1. https://twitter.com/rOpenSci/status/864562357476904960 (12 retweets and 26 likes so far)
2. https://twitter.com/rOpenSci/status/864562646875381760
3. https://twitter.com/rOpenSci/status/864562842351026176